src/wwwroot/app/services/httpService.js

```diff
@@ -1,37 +1,49 @@
 const BASE_URL = 'http://localhost:53031/api/';
 
+let _$http, _$q, _LoggerService;
+
+function _successCallback(response) {
```

Comment:
these functions need to be below class `HttpService`.
All utils function need to write under business-functions